components/badge/badge_button.c

```diff
@@ -27,7 +27,7 @@ void
 badge_button_add(int gpio_num, uint32_t button_id)
 {
 	badge_button_conv[gpio_num] = button_id;
-	badge_button_old_state[gpio_num] = -1;
+	badge_button_old_state[gpio_num] = 1; // released
```

Comment:
it's just the gpio_level. since this is an internal variable and am thinking of rewriting it into a bitmapped state, i think it's ok to leave it this way.